src/escape.rs

```diff
@@ -35,10 +38,12 @@ static HREF_SAFE: [u8; 128] = [
 
 static HEX_CHARS: &'static [u8] = b"0123456789ABCDEF";
 
-pub fn escape_href(ob: &mut String, s: &str) {
+pub fn escape_href(ob: &mut String, s: &str, backslash_escapes: bool) {
```

Comment:
I suspect that this is the wrong place to be special-casing on backslash escaping, that it should have been caught upstream. One way to tell for sure is to look at the AST generated by the [CommonMark testing tool](https://spec.commonmark.org/dingus/).